corehq/apps/accounting/interface.py

```diff
@@ -1024,9 +1025,9 @@ def _invoice_to_row(invoice):
                 contact_info.postal_code,
                 contact_info.country,
                 invoice.account.salesforce_account_id or "--",
-                invoice.date_start,
-                invoice.date_end,
-                invoice.date_due if invoice.date_due else "None",
+                format_datatables_data(invoice.date_start, invoice.date_start),
+                format_datatables_data(invoice.date_end, invoice.date_end),
+                format_datatables_data(invoice.date_due if invoice.date_due else "None", invoice.date_due),
```

Comment:
See other comment -- `format_datatables_data` and the `date_due` conditional are likely unnecessary